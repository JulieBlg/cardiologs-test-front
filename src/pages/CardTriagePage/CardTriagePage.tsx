import React, { FC } from 'react';

const CardTriagePage: FC = () => {
  return <>Hello World !</>; 
}

export default CardTriagePage;
