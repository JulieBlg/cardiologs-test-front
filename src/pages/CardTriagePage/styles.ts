import { Theme } from '@material-ui/core';

const styles = (theme: Theme) => ({
    filtersBar: {
        marginTop: theme.spacing(3),
        marginBottom: theme.spacing(3),
    }
});

export default styles;