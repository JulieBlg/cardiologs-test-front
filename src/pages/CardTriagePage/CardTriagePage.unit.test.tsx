import React from 'react';
import { render } from '@testing-library/react';
import CardTriagePage from './CardTriagePage';

describe('CardTriagePage', () => {
  it('should render CardTriagePage properly', () => {
    const { container } = render(<CardTriagePage />);

    expect(container).toMatchSnapshot();
  });
})

