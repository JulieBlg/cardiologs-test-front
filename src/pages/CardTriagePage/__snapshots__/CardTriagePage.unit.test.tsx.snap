// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTriagePage should render CardTriagePage properly 1`] = `
<div>
  Hello World !
</div>
`;
